lib/tree-view.coffee

```diff
@@ -96,11 +96,16 @@ class TreeView extends View
     @on 'mousedown', '.entry', (e) =>
       @onMouseDown(e)
     @on 'mousedown', '.tree-view-resize-handle', (e) => @resizeStarted(e)
-    @on 'dragstart', '.entry', (e) => @onDragStart(e)
-    @on 'dragenter', '.entry.directory > .header', (e) => @onDragEnter(e)
-    @on 'dragleave', '.entry.directory > .header', (e) => @onDragLeave(e)
-    @on 'dragover', '.entry', (e) => @onDragOver(e)
-    @on 'drop', '.entry', (e) => @onDrop(e)
+    @on 'dragstart', '.entry', (e) =>
+        @onDragStart(e)
+    @on 'dragenter', '.entry.directory > .header', (e) =>
+        @onDragEnter(e)
+    @on 'dragleave', '.entry.directory > .header', (e) =>
+        @onDragLeave(e)
+    @on 'dragover', '.entry', (e) =>
+        @onDragOver(e)
+    @on 'drop', '.entry', (e) =>
+        @onDrop(e)
```

Comment:
It looks like this is just a formatting change. In general with PRs, it's good to avoid extraneous changes like this, in order to ensure that the change merges easily, and also because the maintainers may prefer their existing formatting.